app/main.py

```diff
@@ -1,8 +1,82 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        if not (isinstance(comfort_class, int)
+                and comfort_class in range(1, 8)):
+            raise ValueError("comfort_class argument must be from 1 to 7")
+        self.comfort_class = comfort_class
+
+        if not (isinstance(clean_mark, int)
+                and clean_mark in range(1, 11)):
+            raise ValueError(
+                "clean_mark argument must be from very dirty - 1 to clean - 10"
+            )
+        self.clean_mark = clean_mark
+
+        if not (isinstance(brand, str)):
+            raise TypeError("brand argument is not string")
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int | float,
+            clean_power: int,
+            average_rating: int | float,
+            count_of_ratings: int
+    ) -> None:
+        if not (isinstance(distance_from_city_center, int | float)
+                and distance_from_city_center
+                in [value / 10.0 for value in range(1, 101, 1)]):
```

Comment:
Your function spends extra time creating a list and checking the value inside it. And it can be only float